app/graphql/types/provider.rb

```diff
@@ -9,6 +9,7 @@ class Provider < Types::BaseObject
     field :state, String, null: false
     field :description, String, null: false
     field :requests, [Types::Request], null: false
+    field :responses, Types::Connections::ResponseConnection, connection: true, null: true
```

Comment:
It looks like you don't need the ResponseConnection or ResponseEdge to be added if you're only looking for the count. Instead, let's do:

```
field :response_count, Integer, null: false
def response_count
  object.responses.count
end
```

Reply:
Ah sounds good! I'll make the change.